application-accelerator/provenance.hbs.md

```diff
@@ -21,3 +21,54 @@ project:
 
 For more information about the structure of the file and how to enable application bootstrapping
 provenance, see [Provenance transform](creating-accelerators/transforms/provenance.hbs.md).
+
+## <a id="amr"></a> Integration with AMR
+
+Application Accelerator integrates with [Artifact Metadata Repository](../scst-store/overview.hbs.md) (AMR).
+
+When you generate an application with an accelerator, an event that contains the same information captured by the `Provenance` transform is sent to the AMR store.
+
+The relevant data saved to AMR are:
+
+### <a id='appacceleratorruns'></a> AppAcceleratorRuns (alpha)
+
+The `AppAcceleratorRuns` data model represents new projects running from Git
```

Comment:
I don't understand this sentence.
`AppAcceleratorRun` represents a generation of a project from an accelerator. Whether they get commited to git or not. Also, it's not really "projects running"

Reply:
Would a better wording of this be
"The `AppAcceleratorRuns` data model represents a generat of a project from an accelerator."
?

Reply:
if you meant "generatION" of a project, then yes, this is better

Reply:
Yup! generation is what I meant... must have gotten distracted mid-sentence =)